### ISSUE TITLE
Error replace when pattern has braces

### ISSUE THREAD

**Issue by PFight** [tomaszczechowski/gulp-string-replace#12]
When passing expressin with braces as pattern like this:

    .pipe(replace(`)(WebClient`, `)(/* istanbul ignore next */ WebClient`))

Error throwed:

    SyntaxError: Invalid regular expression: /)(WebClient/: Unmatched ')'

It would be great to see option "noRegex" or smth.

**Comment by tomaszczechowski** [comment]
ok, thanks, investigating and let you know once has feedback.

**Comment by tomaszczechowski** [comment]
Just resolved it. Added `searchValue` options.

```javascript
gulp.task('str-replace', function() {
  var test = '`)(WebClient`, `)(/* istanbul ignore next */ WebClient`';

  gulp.src([ "./test/app.js" ])
    .pipe(replace(test, 'sth', { searchValue: 'string' }))
    .pipe(gulp.dest('./test/result'))
});
```

```bash
[00:23:43] Starting 'str-replace'...
[00:23:43] Finished 'str-replace' after 13 ms
[00:23:43] Starting 'default'...
[00:23:43] Finished 'default' after 12 μs
[00:23:43] Replaced: "`)(WebClient`, `)(/* istanbul ignore next */ WebClient`" to "sth" in a file: app.js
```

- Pls download just released version 1.1.0.
- Pls rate project on github 👍
